frontend/packages/operator-lifecycle-manager/src/dev-catalog.ts

```diff
@@ -23,14 +22,12 @@ export const normalizeClusterServiceVersions = (
   clusterServiceVersions: ClusterServiceVersionKind[],
 ): K8sResourceKind[] => {
   const imgFor = (desc) =>
-    _.get(desc.csv, 'spec.icon')
-      ? `data:${_.get(desc.csv, 'spec.icon', [])[0].mediatype};base64,${
-          _.get(desc.csv, 'spec.icon', [])[0].base64data
-        }`
+    desc.csv?.spec?.icon?.[0]
+      ? `data:${desc.csv.spec.icon[0].mediatype};base64,${desc.csv.spec.icon[0].base64data}`
       : operatorLogo;
 
   const formatTileDescription = (csvDescription: string): string =>
-    `## Operator Description\n${csvDescription}`;
+    `## ${i18n.t('olm~Operator description')}\n${csvDescription}`;
```

Comment:
`Operator Description` heading is on the same line with `csvDescription`, please see below screenshot, this needs to be updated
<img width="899" alt="Screen Shot 2020-12-15 at 3 23 53 PM" src="https://user-images.githubusercontent.com/12692381/102184159-fb56bd80-3ee9-11eb-9d03-999ac4693f2c.png">